map_gui/src/render/building.rs

```diff
@@ -200,7 +188,33 @@ impl DrawBuilding {
                 }
             }
         }
+
+        DrawBuilding {
+            id: bldg.id,
+            label: RefCell::new(None),
+        }
+    }
+
+    pub fn draw_path(
```

Comment:
"path" is a somewhat overloaded term when talking about drawing. Here "path" means something you can walk or drive on, not "path" as in "stroke of a line" (which I initially thought it was).

Take it or leave it, but we could rename this to `draw_driveway`

Reply:
Good point -- renamed building path->driveway everywhere. Once upon a time, it was called the "front path", but that was before cars also used it.